src/main/java/team/unnamed/emojis/EmojisPlugin.java

```diff
@@ -47,6 +47,12 @@
     private ExportService exportService;
     private File database;
 
+    private String messageKick;
+    private String messageWarn;
+    private String messageFailedDownload;
+
+    private boolean packRequired;
+
```

Comment:
It would be better not to have all the messages in the main class, I think it's better to pass the plugin (or plugin configuration) to the listener, and then let the listener get the messages from there

Reply:
If there was ever a reload command wouldn't storing the strings be beneficial?

Reply:
YamlConfiguration already stores those values, don't store it yourself, just use the methods it offers when you need it.

Reply:
It would require re-defining strings, instead of just accessing FileConfiguration values when required